client/src/components/ShoeShowPage.js

```diff
@@ -60,33 +83,74 @@ const ShoeShowPage = ({ user, match }) => {
         getShoe()
     }, [])
 
-    let reviewFormComponent = ""
-    if (user){
-      reviewFormComponent = <ReviewForm postReview={postReview} shoe={shoe} errors={errors} setErrors={setErrors}/>
+  const handleVote = async (value, reviewId) => {
+    try {
+      const response = await fetch(`/api/v1/shoes/${id}/reviews/vote`, {
+        method: "POST",
+        body: JSON.stringify({ value, reviewId }),
```

Comment:
if a vote is for a particular review, then the `reviewId` belongs in the path for the fetch request to follow proper RESTful conventions. We would expect the fetch path to be
```
fetch(`/api/v1/reviews/${reviewId}/votes`)
```
^^ adding the reviewId, `votes` is meant to be plural because we are making a request to interact with the Votes table, and including the shoeId is just extra data - when it comes to creating a new Vote, we don't need to know which shoe it's for.